From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jeannt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jeannt). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
task 343: additional edits and formatting

Lisaco88 committed:
edit pass: app-service-linux-ci-cd

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into app-service-linux-ci-cd

Lisaco88 committed:
edit pass: app-service-linux-ci-cd

Commit:
fix duplicate TOC file issue

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into app-service-linux-ci-cd

Lisaco88 committed:
removing file

Commit:
merge errors

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into app-service-linux-ci-cd

v-viksur committed:
(ForumsTeam) Removed extra characters 

 Removed extra characters

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into app-service-linux-ci-cd

Lisaco88 committed:
edit pass: app-service-linux-ci-cd

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into app-service-linux-ci-cd

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into app-service-linux-ci-cd

shawntabrizi committed:
Updating Azure Service Health Documentation

Adding new name, fixing the resource health and azure status story

shawntabrizi committed:
updating for acrolinx

shawntabrizi committed:
acrolinx update

shawntabrizi committed:
update author

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

v-viksur committed:
(ForumsTeam) Broken Link 

Please review and update the broken link.

shawntabrizi committed:
Update index.md

Juliako committed:
updated but without screenshots

KumudD committed:
remved preview text

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TMGA0313

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TMGA0313

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TMGA0313

v-viinde committed:
USER STORY 1209163 SaaS App Tutorial: Bersin

kadamajay5 committed:
(ForumsTeam)Removed Repeated word"installed"

I have removed the extra word "installed" from Scenario Diagram section.

kadamajay5 committed:
(ForumsTeam) Renamed the link name

Under Windows client section "by using automation" link is taking to TechNet forum thread. I feel either we should change the link with appropriate article/blog/doc or we can change the name of link as by using automation as discussed here. I have changed the link name accordingly. Kindly review it once and do the needful.

shawntabrizi committed:
Removing Personalized from the Brand

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

v-viinde committed:
Updated

v-viinde committed:
Updated

v-viinde committed:
Updated

v-nagta committed:
USER STORY 1210456 SaaS App Tutorial: Amplitude

v-nagta committed:
updated

dlepow committed:
Batch: MPI limit

rojasja committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

rojasja committed:
New Article Custom Event Logger

rojasja committed:
TOC change to add article

dlepow committed:
Fixed link

rojasja committed:
Made spelling corrections and terminology improvements

rojasja committed:
spelling correction considerd

Sogup committed:
Update backup-upgrade-to-vm-backup-stack-v2.md

rojasja committed:
Corrected numbering error

rojasja committed:
small display correction on IMPORTANT

Commit:
toc.yml merger errors

rojasja committed:
correcting IMPORTANT label

KumudD committed:
updated for GA

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TMGA0313

Commit:
task 1035: Add table for Verizon POP abbreviations

tdykstra committed:
clarify usage of output binding with CloudBlobContainer

tdykstra committed:
minor wording change

KumudD committed:
updated dates

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TMGA0313

roygara committed:
Redirecting to PHP dev center

Commit:
task 1035: edits

anrothMSFT committed:
added quotas

roygara committed:
Fixing link.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into bugfix

anrothMSFT committed:
fix table

anrothMSFT committed:
add TPS quotas

anrothMSFT committed:
update TOC

anrothMSFT committed:
missing period

sanjeev3 committed:
Update human-in-the-loop.md

sanjeev3 committed:
Update human-in-the-loop.md

liydu committed:
Make the charge notice as warning

liydu committed:
Change the title

tamram committed:
Update storage-quickstart-blobs-go.md

Lisaco88 committed:
Fixed typo

tdykstra committed:
timer note for function apps sharing storage

tdykstra committed:
note about sharing storage account

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

tdykstra committed:
typo

rojasja committed:
correcting picture

eliminated personal info per guide

PRMerger14 committed:
Merge pull request #35291 from KumudD/TMGA0313

Traffic View and Traffic Real User Measurements

Lisaco88 committed:
Merge pull request #35502 from v-nagta/amplitude

USER STORY 1210456 SaaS App Tutorial: Amplitude

Lisaco88 committed:
Merge pull request #35401 from v-viinde/bersin

USER STORY 1209163 SaaS App Tutorial: Bersin

Ja-Dunn committed:
Update limits-and-quotas.md

casing fix

Ja-Dunn committed:
Update toc.yml

casing fix

PRMerger15 committed:
Merge pull request #35582 from liydu/az3166-remote-monitoring-charge

Make the charge notice as warning

PRMerger17 committed:
Merge pull request #35585 from MicrosoftDocs/tamram-patch-2

Update storage-quickstart-blobs-go.md

Ja-Dunn committed:
Merge pull request #35573 from dksimpson/dean-1035

Dean 1035

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-343

apwestgarth committed:
Update azure-stack-app-service-release-notes-update-one.md

Changes as requested per https://github.com/MicrosoftDocs/azure-docs-pr/pull/35099

Ja-Dunn committed:
Merge pull request #35580 from anrothMSFT/add-quotas

add quotas to documentation

v-geberr committed:
manage account settings

apwestgarth committed:
Update TOC.yml

Changes requested as per https://github.com/MicrosoftDocs/azure-docs-pr/pull/35099

Commit:
quickfix121

PRMerger10 committed:
Merge pull request #35576 from roygara/bugfix

Link is broken due to article being removed, fixing link

v-geberr committed:
edit

Commit:
Task 343: Replace edge server with POP

PRMerger13 committed:
Merge pull request #35590 from MarkusVi/quickfix121

quickfix121

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

nitinme committed:
fixing link

rjagiewich committed:
Merge pull request #35587 from sanjeev3/master

Minor updates to human in the loop article.

tamram committed:
adding information about .NET framework

rjagiewich committed:
Merge pull request #35588 from apwestgarth/patch-4

Update azure-stack-app-service-release-notes-update-one.md

Ja-Dunn committed:
Merge pull request #35589 from v-geberr/0316-account-settings-howto

manage account settings

PRMerger18 committed:
Merge pull request #35594 from dksimpson/dean-343

Dean 343

v-geberr committed:
edits for H1 fix

barbkess committed:
fix links, fixes MicrosoftDocs/azure-docs/#5705

Commit:
Revert "quickfix121"

Commit:
Merge pull request #35600 from MarkusVi/revert-35590-quickfix121

Revert "quickfix121"

Commit:
Merge pull request #35597 from tamram/tamram-0316

adding information about .NET framework

PRMerger10 committed:
Merge pull request #35586 from tdykstra/patch-25

timer note for function apps sharing storage

ShawnJackson committed:
Update app-service-linux-ci-cd.md

Juliako committed:
Update api-management-howto-protect-backend-with-aad.md

ShawnJackson committed:
Merge pull request #34467 from Lisaco88/app-service-linux-ci-cd

edit pass: app-service-linux-ci-cd

Juliako committed:
Update api-management-howto-protect-backend-with-aad.md

BryanTrach-MSFT committed:
(forumsteam) fixes MicrosoftDocs/azure-docs#5842

Updated link to https://docs.microsoft.com/powershell/module/azurerm.apimanagement/?view=azurermps-5.5.0#api_management/ as it has the actual list of API management cmdlets. The original link took users to a Azure news announcement that was only 2 sentences long.

BryanTrach-MSFT committed:
(forumsteam) fixes MicrosoftDocs/azure-docs#5814

Verified that webjobs supports Java webjobs at the two below links. Updating doc to reflect that Java webjobs is supported.

https://docs.microsoft.com/en-us/azure/azure-functions/functions-compare-logic-apps-ms-flow-webjobs#functions-vs-webjobs
https://blogs.msdn.microsoft.com/azureossds/2015/04/28/executing-java-web-jobs-on-azure/

PRMerger14 committed:
Merge pull request #35605 from BryanTrach-MSFT/patch-5

(forumsteam) fixes MicrosoftDocs/azure-docs#5842

PRMerger16 committed:
Merge pull request #35606 from BryanTrach-MSFT/patch-6

(forumsteam) fixes MicrosoftDocs/azure-docs#5814

Juliako committed:
Update TOC.md

PRMerger20 committed:
Merge pull request #35595 from nitinme/nitinme-databricks-updates

Fixing link

shawntabrizi committed:
Updating screenshots for GA

shawntabrizi committed:
small fix with new ui

barbkess committed:
fix checking state in T-SQL, fixes MicrosoftDocs/Azure-docs/#5407

barbkess committed:
fix warning

barbkess committed:
fix warning

PRMerger10 committed:
Merge pull request #35608 from barbkess/git-issues

fix checking state in T-SQL, fixes MicrosoftDocs/Azure-docs/#5407

PRMerger16 committed:
Merge pull request #35599 from v-geberr/0316-fix-h1

edits for H1 fix

gsacavdm committed:
Remove duplicate Q in application-gateway-faqs

PRMerger13 committed:
Merge pull request #35610 from gsacavdm/patch-1

Remove duplicate Q in application-gateway-faqs

gsacavdm committed:
Be consistent about ASC integration for app-gw

PRMerger18 committed:
Merge pull request #35611 from gsacavdm/patch-2

Be consistent about ASC integration for app-gw

PRMerger19 committed:
Merge pull request #35542 from Sogup/patch-39

Update backup-upgrade-to-vm-backup-stack-v2.md

tdykstra committed:
wording change on in/inout types

tdykstra committed:
fix inout types

v-viksur committed:
(ForumsTeam) Typo

Typo

barclayn committed:
adding the entry to the toc that I removed by mistake (I checked the previous PR)

PRMerger13 committed:
Merge pull request #35417 from kadamajay5/patch-3

(ForumsTeam) Renamed the link name

PRMerger14 committed:
Merge pull request #35408 from kadamajay5/patch-1

(ForumsTeam)Removed Repeated word"installed"

femsulu committed:
github issue #5888

added correct url to storage queue error codes

GitHubber17 committed:
Merge pull request #35173 from Juliako/APIMUpdate

updated but without screenshots

PRMerger12 committed:
Merge pull request #35618 from femsulu/patch-3

github issue #5888

ggailey777 committed:
Fixes MicrosoftDocs/azure-docs#5887

GitHubber17 committed:
Merge pull request #35601 from barbkess/load2

fix links, fixes MicrosoftDocs/azure-docs/#5705

GitHubber17 committed:
Merge pull request #35607 from Juliako/patch-3

Update TOC.md

PRMerger17 committed:
Merge pull request #35619 from ggailey777/patch-15

Fixed an anchor link to the removed Scheduler section

femsulu committed:
(Forums Team) Github issue #5933

Replaced "understading" with "Understanding" in doc url.

GitHubber17 committed:
Merge pull request #35616 from barclayn/siem-toc

adding the entry to the toc that I removed by mistake (I checked the …

MicrosoftGuyJFlo committed:
[MFA] Fix link in configure settings doc

Resolves https://github.com/MicrosoftDocs/azure-docs/issues/5879

Commit:
Merge pull request #35620 from femsulu/patch-4

(Forums Team) Github issue #5933

PRMerger10 committed:
Merge pull request #35621 from MicrosoftGuyJFlo/patch-6

[MFA] Fix link in configure settings doc

KumudD committed:
removed preview text

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TMGA0313

PRMerger10 committed:
Merge pull request #35622 from KumudD/TMGA0313

FAQ updates

KumudD committed:
tweak

klaaslanghout committed:
we no longer have the need to do batch delete

Juliako committed:
Update api-management-navigate-to-instance.md

PRMerger10 committed:
Merge pull request #35628 from Juliako/patch-5

Update api-management-navigate-to-instance.md

snehaamicrosoft committed:
string change and reordering of items

Commit:
Merge pull request #35624 from KumudD/TMGA0318

Traffic Manager FAQ update

Commit:
Updated answer for question 14

carolinacmoravia committed:
C12405: Broken layout due to missing breaklines

Hello, @Barclayn, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version. Also, source version displays with broken layout.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

v-geberr committed:
luis website callout

Commit:
automation: metadata cleanup

Commit:
Merge pull request #35633 from jimdial/patch-96

Updated answer for question 14

Commit:
Merge pull request #35634 from carolinacmoravia/patch-651

C12405: Broken layout due to missing breaklines

Commit:
Merge pull request #35636 from v-geberr/0319-fix-region-reference

luis website callout on regions reference

Commit:
Merge pull request #35637 from georgewallace/autometadata

automation: metadata cleanup

Commit:
automation: metadata cleanup #2

v-geberr committed:
bot tutorial next step speech priming

v-geberr committed:
more coffee needed

PRMerger17 committed:
Merge pull request #35641 from v-geberr/0319-bot-tutorial-speech-priming

0319 bot tutorial speech priming

tdykstra committed:
remove unused image file

itechedit committed:
Merge pull request #35039 from shawntabrizi/master

Updating Azure Service Health Documentation

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

tdykstra committed:
clarify binding direction = direction property

dlepow committed:
fixes per MarkScu

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge branch 'FromPublicRepo' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPublicRepo

PRMerger15 committed:
Merge pull request #35638 from georgewallace/autometadata

automation: metadata cleanup #2

Commit:
automation: metadata cleanup #3

PRMerger16 committed:
Merge pull request #35626 from klaaslanghout/patch-18

we no longer have the need to do batch delete

davidmu1 committed:
git issue 5944

rmca14 committed:
Merge pull request #35632 from snehaamicrosoft/migrate

string change and reordering of items

CarlRabeler committed:
Update az-overview.md

douglaslMS committed:
Added info about data retention per GitHub issue.

Commit:
Merge pull request #35651 from davidmu1/gitissue5944

git issue 5944

Commit:
Merge pull request #35533 from dlepow/fix

Batch: MPI limit

PRMerger10 committed:
Merge pull request #35654 from douglaslMS/adf-run-retention

Added info about data retention per GitHub issue.

PRMerger13 committed:
Merge pull request #35615 from v-viksur/patch-28

(ForumsTeam) Typo

PRMerger13 committed:
Merge pull request #34877 from v-viksur/patch-24

(ForumsTeam) Removed extra characters

PRMerger14 committed:
Merge pull request #35098 from v-viksur/patch-27

(ForumsTeam) Broken Link

PRMerger16 committed:
Merge pull request #35571 from tdykstra/patch-24

clarify usage of output binding with CloudBlobContainer

PRMerger16 committed:
Merge pull request #35648 from georgewallace/autometadata

automation: metadata cleanup #3

v-shils committed:
Merge pull request #35643 from tdykstra/delcs

remove unused image file

GitHubber17 committed:
Merge pull request #35540 from rojasja/master

Master

v-alje committed:
Merge pull request #35647 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #35650 from CarlRabeler/patch-30

Update az-overview.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
